app/features/send/send-store.ts

```diff
@@ -11,6 +11,18 @@ import { type Contact, isContact } from '../contacts/contact';
 import { DomainError } from '../shared/error';
 import type { CashuLightningQuote } from './cashu-send-quote-service';
 import type { CashuSwapQuote } from './cashu-send-swap-service';
+import type { SparkLightningQuote } from './spark-send-quote-service';
+
+/**
+ * Returns the default send type based on account type.
+ * - Cashu accounts default to CASHU_TOKEN (no destination required)
+ * - Spark accounts default to BOLT11_INVOICE (destination required)
+ */
+const getDefaultSendType = (
+  accountType: Account['type'],
+): 'CASHU_TOKEN' | 'BOLT11_INVOICE' => {
```

Comment:
instead of repeating the union type here the return type of this should be SendType